### PR TITLE
Add Soverin to providers

### PR DESCRIPTION
**Pull request by GM-Polyakov**
Added settings for soverin.net email provider to autodetected providers.

### CODE REVIEW

File: inbox/providers.py [hunk 1]
```diff
@@ -141,6 +141,14 @@ def provider_info(provider_name):
         "domains": ["icloud.com"],
         "mx_servers": ["mx[1-6].mail.icloud.com"]
     }),
+    ("soverin", {
+        "type": "generic",
+        "imap": ("imap.soverin.net", 143),
```
Comment (khamidou) [inline inbox/providers.py]: The port should be 993. 143 is IMAP w/ STARTTLS, which is slightly less secure.

**Comment by khamidou** [comment]
LGTM minus my inline comment.

File: inbox/providers.py (hunk 1 above)
Comment (GM-Polyakov) [inline inbox/providers.py]: https://github.com/nylas/sync-engine/commit/f4cf6839ff867722a5e33015a4d2e24dd5969283

**Comment by khamidou** [comment]
Thanks for fixing!

**Pull request merged**